recipe/ff_ci_pr_build.py

```diff
@@ -61,7 +61,12 @@ def circle_check_latest_pr_build(repo, pr, build_num):
     builds = request_json(url.format(repo=repo, pr=pr), headers=headers)
 
     # Parse the response to get a list of build numbers for this PR.
-    pr_build_nums = set(map(lambda b: int(b["build_num"]), builds))
+    job_name = os.environ.get("CIRCLE_JOB")
+    same_param_builds = list(filter(
+        lambda b: b.get("build_parameters", {}).get("CIRCLE_JOB") == job_name,
```

Comment:
For some reason, the default value of this `get` method for `"build_parameters"` is not being used. 😕 Not entirely sure why that is happening, but it does happen to me locally as well. Suggestions welcome.